assets/styles/_responsive.scss

```diff
@@ -143,12 +154,14 @@
     }
 
     .slider-overlay-container {
-      margin-top: -99vh;
+      margin-top: -110vh;
```

Comment:
let's refrain from changing heights and widths from now. let's focus on the fonts and typography.